dev-system/resources/kms_examples/granuledataformat

```diff
@@ -0,0 +1,6 @@
+"Keyword Version: draft","Revision: 2019-11-01 11:34:56","Timestamp: 2019-11-05 12:52:55","Terms Of Use: http://gcmd.nasa.gov/r/l/TermsOfUse","The most up to date XML representations can be found here: https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/GranuleDataFormat/?format=xml""Case native"
```

Comment:
The example says https://gcmdservices.gsfc.nasa.gov/kms/concepts/concept_scheme/GranuleDataFormat/?format=xml"

but GranuleDataFormat should be granuledataformat